frontend_tests/node_tests/popovers.js

```diff
@@ -171,6 +171,7 @@ run_test("sender_hover", (override) => {
                     is_active: true,
                     is_bot: undefined,
                     is_sender_popover: true,
+                    has_message_context: true,
```

Comment:
I think we actually do want "Reply mentioning user" in the right sidebar popovers, though -- it's useful when you find a user in the right sidebar and want to generate mention syntax for them.  I suppose we could rename to "Copy mention syntax" when `!has_message_context` to address that use case without the weird connection to a message that's selected in a totally different part of the UI?